packages/desktop-client/src/components/common/View.tsx

```diff
@@ -22,7 +22,7 @@ const View = (props: ViewProps) => {
       {...restProps}
       ref={innerRef}
       style={nativeStyle}
-      className={`view ${css([props.className, props.style])}`}
+      className={`view ${props.className} ${css([props.style])}`}
```

Comment:
`className` was not working before :)

Reply:
Maybe this was the root cause of the issue I was having when I experimented with emotionjs.